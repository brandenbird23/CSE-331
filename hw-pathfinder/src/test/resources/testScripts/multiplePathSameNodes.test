# Test for when multiple paths for same node

CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B 7.5
AddEdge graph A B 8.0

FindPath graph A B