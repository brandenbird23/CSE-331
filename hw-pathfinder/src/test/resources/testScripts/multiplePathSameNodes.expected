# Test for when multiple paths for same node

created graph graph

added node A to graph
added node B to graph

added edge 7.500 from A to B in graph
added edge 8.000 from A to B in graph

path from A to B:
A to B with weight 7.500
total cost: 7.500